builders/scripts/builder_darwin_amd64_runtime

```diff
@@ -7,11 +9,124 @@ cd "/root/runtime/lib/"
 cp -d /root/osxcross/target/macports/pkgs/opt/local/lib/*.dylib "/root/runtime/lib"
 rm /root/runtime/lib/libpython2.7.dylib
 
+## Remove all the XQuartz files, not needed and shrink file size!
```

Comment:
Why aren't these file needed?

Reply:
One XQuarts uses those files, and since we don't bundled a version they wont be needed.
These are the exact files I've since removed from my version of Wineskin without issue